Pipeline/routes/route.py

```diff
@@ -109,9 +109,23 @@ def get_graphs_per_module():
         abort(400, "Failed to retrieve representative graphs. Please check your inputs. Error: " + str(e))
 
 
-@routes.route('/rnamigos', methods=['GET'])
-def greeting_rna_migos():
-    return RnaMigos.greeting()
+@routes.route('/rnamigos_string', methods=['POST'])
+def rnamigos_string():
+    '''
+    Represents the RnaMigos endpoint for string input.
+
+    :returns: RnaMigos BayesPairing output
+    '''
+    try:
+        if not request.form or "graphs" not in request.form:
+            abort(400, description="Did not receive any arguments.")
+        result = RnaMigos.rnamigos_string(request.form.to_dict())
+        if (result):
+            return result.json()
+
+        abort(result.status_code, result.json().get("error"))
+    except Exception as e:
+        abort(400, "RnaMigos failed to process data. Please check your inputs or BayesPairing. Error: " + str(e))
```

Comment:
I find it kinda redundant to have if conditions with abort within a try block, cuz like if it fails it's gonna abort anyways. If you wanted specific error messages for each type of error i guess it could be chill. I still think there's a better way you could structure it though

Reply:
tbh I didn't write that part of the code so I'm not sure, but I do agree that we should standardize our error handling/logging capabilities throughout our pipeline and services. I'll also bring that up in our group chat. Thanks!